Ginger/GingerCoreNET/Run/Excels/ExcelNPOIOperations.cs

```diff
@@ -354,5 +353,13 @@ public List<string> GetSheets(string fileName)
             }
             return sheets.OrderBy(itm => itm).ToList();
         }
+
+        public void Dispose()
+        {
+            mSheet = null;
+            mWorkbook.Close();
+            mWorkbook = null;
+            GC.Collect();
```

Comment:
try to comment GC.collect() and see if it will still work. 
If we are disposing the objects  then garbage collector is automatically called after some interval.